src/core/ddsi/src/ddsi_plist.c

```diff
@@ -1761,11 +1761,9 @@ static const struct piddesc piddesc_eclipse[] = {
 #ifdef DDS_HAS_TYPE_DISCOVERY
   QP  (CYCLONE_TYPE_INFORMATION,         type_information, XO),
 #endif
-#ifdef DDS_HAS_SHM
-    { PID_PAD, PDF_QOS, QP_SHARED_MEMORY, "CYCLONE_SHARED_MEMORY",
-    offsetof(struct ddsi_plist, qos.shared_memory), membersize(struct ddsi_plist, qos.shared_memory),
-    {.desc = { Xb, XSTOP } }, 0 },
-#endif
+    { PID_PAD, PDF_QOS, QP_LOCATOR_MASK, "CYCLONE_LOCATOR_MASK",
+    offsetof(struct ddsi_plist, qos.ignore_locator_type), membersize(struct ddsi_plist, qos.ignore_locator_type),
+    {.desc = { Xi, XSTOP } }, 0 },
```

Comment:
Changing the bitmask type to a `uint32_t` means this should be `Xu`.

Reply:
Yep.